File: SEP6/primeNum.py
def isPrime(num):
    numDivisors = 0
    for i in range(1,num+1):
        if ( num % i == 0):
            numDivisors = numDivisors + 1
        if ( numDivisors > 2 ):
            return False

def printPrime(num1, num2):
    for i in range (num1, num2+1):
        if (isPrime(i) != False):
            print (i)
def main():
    val1 = int(input ("Enter a value:"))
    val2 = int(input ("Enter a value:"))
    printPrime(val1, val2)

main()
    

File: SEP13/PrimeNum.c
#include <stdio.h>
void main() {
    int num, i;
    printf("Enter a number: ");
    scanf ("%d", &num);
    printPrime(num);
}
int isPrime(int num){
    int numDivisors = 0, i;
    for (i = 1; i < num+1; i++){
        if (num%i == 0){
            numDivisors++;
        }
        if (numDivisors > 2){
            return 0;
        }

    }
    return 1;

}
void printPrime(int num){
    int i;
    for (i = 2; i< num+1; i++){
        if (isPrime(i) != 0){
            printf("%d", i);
            printf("\n");
        }
        
    }
}